docs/knowledge_base/tips/skippable_tasks/index.md

```diff
@@ -0,0 +1,208 @@
+...
+```
+
+With the code below, we create and submit an instance of this scenario configuration.
+
+```python linenums="1"
+scenario = tp.create_scenario(scenario_cfg)
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario.sum.read())
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_4.png){width=100%}
+
+The task associated with my Task has been completed, which means that my function has been executed.
+
+The line below is resubmitting the scenario, but please note that I haven't made 
+any changes to my input Data nodes in any way.
+
+```python linenums="1"
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;,scenario.sum.read())
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_5.png){width=100%}
+
+As expected, Taipy is skipping the Task because the input parameters haven't changed. 
+If there are multiple tasks in this scenario, Taipy may skip several of them.
+
+The code below shows what happens when we submit the scenario after making a change to an input Data node. 
+In this case, the value of **nb_1** is updated from 21 to 42.
+
+```python linenums="1"
+scenario.nb_1.write(42)
+tp.submit(scenario)
+print(&amp;amp;amp;quot;Results (sum):&amp;amp;amp;quot;, scenario.sum.read())
+```
+
+![Leveraging Skippability in Taipy Tasks](skippable_tasks_6.png){width=100%}
+
+The input changed, so Taipy will re-execute my Task and give the appropriate results (44).
+
+## Using Global Data Nodes
+
+Skipping tasks doesn't only happen when you resubmit a scenario or pipeline; 
+it can also occur when creating and submitting a completely new scenario with Global Data nodes.
+
+For instance, if you want to preprocess a raw data set and make the result accessible globally 
+across the entire application, you can change the scope of both Data nodes (the raw data set and the result) to Global. 
+This means that all scenarios will share these Data nodes, and the Task related to this operation might be 
+skipped across different scenarios.
+
+Let's revisit our previous code and modify the Data nodes to have a Global scope.
+
+<iframe width="640" height="360" src="https://www.taipy.io/wp-content/uploads/2023/03/predict.mp4?_=3" frameborder="0" allowfullscreen></iframe>
+
+```python linenums="1"
+from taipy.config import Config
+model_cfg = Config.configure_data_node(&amp;amp;amp;quot;model&amp;amp;amp;quot;, 
+                                       default_path=&amp;amp;amp;quot;model.p&amp;amp;amp;quot;)
+predictions_cfg = Config.configure_data_node(&amp;amp;amp;quot;predictions&amp;amp;amp;quot;)
+task_cfg = Config.configure_task(&amp;amp;amp;quot;task&amp;amp;amp;quot;,
+                              predict,
+                              model_cfg,
+                              predictions_cfg)
+scenario_cfg = Config.configure_scenario_from_tasks(&amp;amp;amp;quot;scenario&amp;amp;amp;quot;, [task_cfg])
```

Comment:
Same remark on the string code that is messed up + same remark on configure_scenario_from_tasks